app/views/surveys/_form.html.erb

```diff
@@ -0,0 +1,75 @@
+<%= fields_for :survey, survey do |f|  %>
+  <div class="row">
+    <div class="col">
+      <div class="form-group">
+        <label for="description">Titel (deutsch)</label>
+        <%= f.text_field :title_de, name: "survey[title][de]", class: "form-control" %>
+      </div>
+      <div class="form-group">
+        <label for="description">Titel (polnisch)</label>
+        <%= f.text_field :title_pl, name: "survey[title][pl]", class: "form-control" %>
+      </div>
+    </div>
+  </div>
+
+  <div class="row">
+    <div class="col">
+      <div class="form-group">
+        <label for="description">Beschreibung (deutsch)</label>
+        <%= f.text_area :description_de, name: "survey[description][de]", class: "form-control", rows: 3 %>
+      </div>
+      <div class="form-group">
+        <label for="description">Beschreibung (polnisch)</label>
+        <%= f.text_area :description_pl, name: "survey[description][pl]", class: "form-control", rows: 3 %>
+      </div>
+    </div>
+  </div>
+
+  <div class="row">
+    <div class="col">
+      <div class="form-group">
+        <label for="description">Frage (deutsch)</label>
+        <%= f.text_field :question_title_de, name: "survey[question_title][de]", class: "form-control" %>
```

Comment:
required missing (thx to @digorath)

Reply:
will be changed to:

![Bildschirmfoto 2021-06-21 um 11 26 48](https://user-images.githubusercontent.com/1942953/122739852-ab372a00-d283-11eb-821a-118d2c257a8d.png)

validation is done by html5 in the frontend. we do not have a validation per application. (this is one big downside of the whole architecture of the cms)

date fields were already set as required.